docs/cookbook.md

```diff
@@ -237,7 +237,7 @@ obs.put(store2, path2, Path("temporary_file"))
 It's easy to **stream** a download from one store directly as the upload to another. Only the given
 
 !!! note
-Using the async API is currently required to use streaming copies.
+    Using the async API is currently required to use streaming copies.
```

Comment:
The vscode auto formatter for markdown doesn't keep this indented 😕

Reply:
Try adding a newline before. ( MkDocs will still format it correctly.)